site/docs/flink.md

```diff
@@ -18,24 +18,24 @@
 # Flink
 
 Apache Iceberg supports both [Apache Flink](https://flink.apache.org/)'s DataStream API and Table API to write records into an Iceberg table. Currently,
-we only integrate Iceberg with Apache Flink 1.11.x.
-
-| Feature support                                                        |  Flink 1.11.0      |  Notes                                                 |
-|------------------------------------------------------------------------|--------------------|--------------------------------------------------------|
-| [SQL create catalog](#creating-catalogs-and-using-catalogs)            | ✔️                 |                                                        |
-| [SQL create database](#create-database)                                | ✔️                 |                                                        |
-| [SQL create table](#create-table)                                      | ✔️                 |                                                        |
-| [SQL create table like](#create-table-like)                            | ✔️                 |                                                        |
-| [SQL alter table](#alter-table)                                        | ✔️                 | Only support altering table properties, Columns/PartitionKey changes are not supported now|
-| [SQL drop_table](#drop-table)                                          | ✔️                 |                                                        |
-| [SQL select](#querying-with-sql)                                       | ✔️                 | Support both streaming and batch mode                  |
-| [SQL insert into](#insert-into)                                        | ✔️ ️               | Support both streaming and batch mode                  |
-| [SQL insert overwrite](#insert-overwrite)                              | ✔️ ️               |                                                        |
-| [DataStream read](#reading-with-datastream)                            | ✔️ ️               |                                                        |
-| [DataStream append](#appending-data)                                   | ✔️ ️               |                                                        |
-| [DataStream overwrite](#overwrite-data)                                | ✔️ ️               |                                                        |
-| [Metadata tables](#inspecting-tables)                                  |    ️               | Support Java API but does not support Flink SQL        |
-| [Rewrite files action](#rewrite-files-action)                          | ✔️ ️               |                                                        |
+we only integrate Iceberg with Apache Flink 1.12.x\1.13.x\1.14.x
```

Comment:
Please fix this list. List items in documentation should be separated by `, ` and should use "and" or "or" before the last item.

In addition, please remove personal pronouns, like "we". Documentation should be direct and state what Iceberg does. Using "we" is confusing because it is not clear who it is referred to.

This sentence should be: "Iceberg integration for Apache Flink is available for Flink versions 1.12, 1.13, and 1.14. Previous versions of Iceberg also support Flink 1.11."

Reply:
I updated this.

Reply:
thanks